content/blog/1.0-announcement/index.mdx

```diff
@@ -0,0 +1,73 @@
+
+  
+
+Versioned and Immutable is often understood as “use git” but there is more to it than that. If that desired state describes using a “latest” tag for example, it is no longer versioned because there is no way to rollback changes in the desired state. Likewise, many version control systems can be used in GitOps as long as they meet those two basic requirements and teams use them in a conformant manner.
+
+  
+
+Pulled Automatically means we have to have software agents constantly observing the desired state. While triggers may exist to speed up that observation, a GitOps system shouldn’t exclusively rely on them. Software agents need to be smart enough to constantly be aware of what the desired state should be. The verb “pull” here is used very clearly in contrast to the way traditional CD pipelines function based on triggers that simply push.
+
+  
+
+Finally, Continuously Reconciling brings it all together. The GitOps software agents have to be aware of the actual state of a system under management and attempt to apply the desired state. Being constantly aware of both the actual state and desired state means we can detect anytime they are out of alignment whether by changes to the desired state (normal operation) or changes to the actual state (drift), a GitOps software agent should detect this attempt to apply the desired state.
+
+  
+
+When teams implement these principles and achieve GitOps they deploy more often, have fewer regressions, and are more competitive. They bring together decades of experience delivering software to create a standard that is incredibly powerful and accessible and rapidly being adopted by a growing number of open source tools and vendors.
+
+## What’s next
+
+With the publication of this standard we’re announcing two new initiatives to help more and more teams adopt these principles. First, “GitOps Blueprints” will be a place where anyone from the community can bring their stacks and share how they’ve implemented GitOps using various tools. Want to know how others have implemented GitOps using your favorite cloud, SCM, and orchestrator? GitOps Blueprints is a place where you’ll be able to see examples from people implementing GitOps in practice.
+
+  
+
+Second, “GitOps Whitepapers” is a place to explore ideas around implementing the GitOps principles. For example, what if your SCM system becomes unavailable? When is it ok to “break glass” and start making manual changes? GitOps Whitepapers are meant to be a space where the community can share these ideas and document them.
+
+  
+
+There’s so much more on the horizon and we need your help! [Get involved](https://opengitops.dev/get-involved)! We’d love more participation to build GitOps Blueprints, GitOps Whitepapers, plan GitOps events, and help with the direction of the Open GitOps projects.
```

Comment:
```suggestion
There’s so much more on the horizon and we need your help! [Get involved](https://opengitops.dev/get-involved)! We’d love more participation to build GitOps Blueprints, GitOps Green Papers, plan GitOps events, and help with the direction of the Open GitOps project.
```
See earlier re: green papers.

Coincidentally, Blue-Green deployments is a terminology in Software Engineering.  If we need a colorful mnemonic, this "GitOps Blueprints", "GitOps Green Papers", and Blue-Green deployments can be combined toward meaningful outcomes.